modelopt/torch/speculative/plugins/transformers.py

```diff
@@ -311,7 +311,7 @@ def forward(
                 hidden_states,
                 attention_mask=attention_mask,
                 position_ids=position_ids,
-                past_key_value=past_key_values,
+                past_key_values=past_key_values,
```

Comment:
I think llamadecoderlayer takes `past_key_value` instead of `past_key_values`.
https://github.com/huggingface/transformers/blob/v4.55-release/src/transformers/models/llama/modeling_llama.py#L280
Is this a typo?

Reply:
I see. Please use the new version then (`past_key_values`). We can update the version requirement later.